src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2085 @@
+# New Tracker
+
+Version 2.36 of DHIS2 introduced a set of brand new tracker endpoints dedicated to importing and querying tracker objects (Including tracked entities, enrollments, events, and relationships).
+These new endpoints set a discontinuity with earlier implementations. Re-engineering the endpoints allowed developers to improve, redesign, and formalize the API's behavior to improve the Tracker services.
+
+The newly introduced endpoints consist of:
+* `POST /api/tracker`
+* `GET /api/enrollments`
+* `GET /api/events`
+* `GET /api/trackedEntities`
+* `GET /api/relationships`
```

Comment:
This sounds  a bit ambiguous - only the /api/tracker is new from this list. Did you mean `/api/tracker/..`?

Reply:
Correct